R/get.R

```diff
@@ -526,6 +526,72 @@ get_usage_static <- function(
   return(out)
 }
 
+#' Get usage information for deployed content
+#'
+#' @description
+
+#' Retrieve content hits for all available content on the server. Available
+#' content depends on the user whose API key is in use. Administrator accounts
+#' will receive data for all content on the server. Publishers will receive data
+#' for all content they own or collaborate on.
+#'
+#' If no date-times are provided, all usage data will be returned.
+
+#' @param client A `Connect` R6 client object.
+#' @param from Optional `Date` or date-time (`POSIXct` or `POSIXlt`). Only
```

Comment:
If it's optional, what happens if I omit it?

Reply:
Why am I having so much trouble coming up with the language for this? 😂 

If it's omitted, the time constraint is unbounded on the lower side.